test/common/odp_cunit_common.c

```diff
@@ -273,6 +273,40 @@ static int default_term_func(void)
 	return odp_cunit_print_inactive();
 }
 
+static void _cunit_test_setup_func(void)
+{
+	CU_AllTestsCompleteMessageHandler all_test_comp_handler;
+	CU_SuiteCompleteMessageHandler suite_comp_handler;
+	CU_pFailureRecord pfailrec;
+	CU_pSuite psuite;
```

Comment:
A bit nitpicky, but `psuite` could be renamed to `suite` to be more consistent with the rest of the file.